### PR TITLE
Add Python solutions to CSES 1073 (Towers)

### PR DESCRIPTION
**Pull request by jaredliw**
_If any of the below doesn't apply to this Pull Request, mark the checkbox as
done._

- [x] I have tested my code
- [x] I have followed the code style guidelines mentioned
      [here](https://usaco.guide/general/adding-solution/#code-conventions)
- [x] I have properly formatted the files according to the steps mentioned
      [here](https://usaco.guide/general/adding-solution#steps)

### CODE REVIEW

File: solutions/cses-1073.mdx [hunk 1]
```diff
@@ -108,6 +108,39 @@ public class Towers {
...
+		else:
+			right = mid - 1
+			tower_idx = mid
+
+	# If there exists a satisfying tower, add the cube to that tower
+	# and update the top element of the tower
+	if tower_idx != -1:
+		towers[tower_idx] = cube
+	# If there aren't any suitable towers, append new tower to end of
+	# array
```
Comment (SansPapyrus683) [inline solutions/cses-1073.mdx:136]: and yeah, no need for the 2 lines here(?)

File: solutions/cses-1073.mdx [hunk 2]
```diff
@@ -108,6 +108,39 @@ public class Towers {
 ```
 </JavaSection>
 
+<PySection>
+
+```python
+input()
+# Store the topmost cube of each tower
+# Towers are in non-decreasing order for sure
```
Comment (SansPapyrus683) [inline solutions/cses-1073.mdx:116]: this was proven in the editorial right

**Review by SansPapyrus683 — CHANGES_REQUESTED** [review]

File: solutions/cses-1073.mdx (hunk 1 above)
Comment (jaredliw) [reply solutions/cses-1073.mdx:136]: (line breaks are done manually by feeling lol)

File: solutions/cses-1073.mdx (hunk 2 above)
Comment (SansPapyrus683) [reply solutions/cses-1073.mdx]: uh was it?

File: solutions/cses-1073.mdx (hunk 1 above)
Comment (SansPapyrus683) [reply solutions/cses-1073.mdx]: oh what text editor you using?

File: solutions/cses-1073.mdx (hunk 2 above)
Comment (jaredliw) [reply solutions/cses-1073.mdx]: > ### ... If no such tower exists, append a new tower to the end of the array. By doing so, we maintain the tower array in a sorted order (try and prove this for yourself).

yes......?

File: solutions/cses-1073.mdx (hunk 1 above)
Comment (jaredliw) [reply solutions/cses-1073.mdx]: ~~notepad in most of the time~~
I have sublime text as well (ik there is a plugin that do the stuff)

File: solutions/cses-1073.mdx (hunk 2 above)
Comment (SansPapyrus683) [reply solutions/cses-1073.mdx]: alr thx

**Pull request merged**